start-kind.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+CONTAINERS="tigrisdata/tigris:1.0.0-beta.18
+tigrisdata/typesense:latest
+tigrisdata/ts-node-mgr:latest
+foundationdb/foundationdb:7.1.7
+foundationdb/foundationdb-kubernetes-sidecar:7.1.7-1
+grafana/grafana:latest
+busybox:latest
+victoriametrics/victoria-metrics:latest
+foundationdb/fdb-kubernetes-operator:v1.6.0
+"
+
+for container in ${CONTAINERS}
+do
+  echo "Pulling ${container}"
+  docker pull ${container}
+done
+
+kind create cluster --config kind-config.yaml --image kindest/node:v1.21.14
```

Comment:
Consider capturing kind version as a variable. Makes overrides from CI easier in the future.

Reply:
Makes sense, did that.